app/components/projects/prebuilds/archive-button.tsx

```diff
@@ -0,0 +1,26 @@
+import { FormButtonWithModal } from "~/components/form-button-with-modal";
+import { PagePaths } from "~/page-paths.shared";
+
+export function PrebuildArchiveButton(props: { prebuildExternalId: string }): JSX.Element {
+  return (
+    <FormButtonWithModal
+      action={PagePaths.ArchivePrebuild}
+      method="POST"
+      inputs={{
+        prebuildExternalId: props.prebuildExternalId,
+      }}
+      buttonProps={{ color: "light", className: "transition-all" }}
+      loadingSpinnerProps={{ color: "gray" }}
+      modal={{
+        header: "Archive Prebuild",
+        body: "Are you sure you want to archive this prebuild? Its backing files will be deleted and it will no longer be possible to use it to create new workspaces.",
```

Comment:
```suggestion
        body: "Are you sure you want to archive this prebuild? It's backing files will be deleted and it will no longer be possible to use it to create new workspaces.",
```

Reply:
https://www.dictionary.com/e/its-vs-its/